io/ntriples/common/src/main/scala/org/w3/banana/io/NTriplesWriter.scala

```diff
@@ -24,26 +27,35 @@ class NTriplesWriter[Rdf <: RDF](implicit val ops:RDFOps[Rdf]) extends RDFWriter
    * @param node Rdf node
    * @return
    */
-  def node2Str(node:Rdf#Node): String = ops.foldNode(node)(
-    { case ops.URI(url) => "<" + url + ">"},
-    { case ops.BNode(id) => "_:" + id},
-    {
-      case ops.Literal((string, ops.URI(datatype), Some(lang))) => "\"" + string + "\"" + "@" + lang
-      case ops.Literal((string, ops.URI(datatype), None)) => "\"" + string + "\"" + "^^<" + datatype + ">"
+  def node2Str(node: Rdf#Node, base: Rdf#URI): String = ops.foldNode(node)(
+    uri => "<" + asString(uri, base) + ">",
```

Comment:
Does this ever do anything with the base? Looks like always just the URI is returned. Also, I can't see a test that tests a base URI. Maybe I'm missing something...